README.md

```diff
@@ -561,6 +574,68 @@ module.exports = function(server) {
 };
 ```
 
+# Mass update
+
+Given an array of data to be updated, cloudant supports the idea of performing bulk update on a model instance.
+
+**Note:** To perform mass update, each data in the array data set needs to have a `_rev` property that matches the corresponding documents `_rev` property in the database.
```

Comment:
Given my other feedback, it's also not the case that `_rev` is required; it's only necessary if you know the record exists and want to replace it.